### DISCUSSION TITLE
Variables to be removed

### DISCUSSION THREAD

**Discussion by twitte01** [twitte01/232R_GroupProject#51] [Miscellaneous]
Variables that should be removed or edited

**Comment by twitte01** [comment]
**World Happiness Dataset**

- investigate rank in 2012
- may want to try putting changes in context of other countries or western counties (could be general range or avg shifts within country)

**Comment by twitte01** [comment]
investigate what household weights means and how it should impact clustering

**Comment by twitte01** [comment]
**Individual Variables**

- investigate how person weight should be used or should be excluded
- exclude health insurance coverage vs not for the more detailed insurance variable
- how should we represent missing 
- used detailed employment status but recode
- exclude (technical variables beside year)

**Reply by yudon90** [reply]
-investigate how person weight should be used or should be excluded:
Keeping PERWT due to it helps the analysis to be representative of the broader population. It helps correct for biases and provides more accurate insights. May be used in conjunction with HHWT to help normalize the data for the broader population

-exclude health insurance coverage vs not for the more detailed insurance variable:
Decided to keep both general and detailed insurance variable. It might be interesting to see if the different types of health coverage may affect the model.

-how should we represent missing:
Based off the data exploration and some of my own exploration, I couldn't find any missing data. I think we are good here unless anyone finds anything.

-used detailed employment status but recode:
Added new column for detailed employment status as string. Also, performed additional data exploration.


-exclude (technical variables beside year)
Removed the following variables:('SAMPLE', 'SERIAL', 'CBSERIAL', 'HHWT', 'PERNUM', 'CBPERNUM', 'CLUSTER', 'CPI99', 'STRATA', 'EMPSTAT', 'CLASSWKR')

**Comment by twitte01** [comment]
**Household Weights**

- simplify household types with gender 
- investigate multiple entries for household that are not duplicates 
- combine housing unit types to simplify & farm variable 
- look at state distribution with population distribution in those states
- deal with access to internet N/a
- exclude couple type due to limited years
- how should we deal with ownership? 
- exclude technical variables besides year & HHtype
